fc/verify.go

```diff
@@ -16,23 +16,26 @@ type VerifySignalsResponse struct {
 }
 
 type VerifyMatchResponse struct {
-	City       bool `json:"city"`
-	Region     bool `json:"region"`
-	Country    bool `json:"country"`
-	Continent  bool `json:"continent"`
-	PostalCode bool `json:"postalCode"`
-	FamilyName bool `json:"familyName"`
-	GivenName  bool `json:"givenName"`
-	Phone      bool `json:"phone"`
-	Email      bool `json:"email"`
-	Maid       bool `json:"maid"`
-	Social     bool `json:"social"`
-	NonId      bool `json:"nonId"`
+	City       bool    `json:"city"`
+	Region     bool    `json:"region"`
+	Country    bool    `json:"country"`
+	Continent  bool    `json:"continent"`
```

Comment:
For verify match, the response in doc is
`{  
    "city" : "household",
    "region" : "household",
    "country" : "household",
    "postalCode" : "household",
    "givenName" : "unknown",
    "familyName": "household",
    "phone" : "tangled",
    "email" : "self",
    "risk" : 0.21
}`

We currently have some additional fields such as NonId, Continent etc. 
Were these fields removed in new version or are the missing from the document?
https://docs.fullcontact.com/docs/verify-match

Reply:
removed all unavailable fields